code/game/objects/items/granters.dm

```diff
@@ -439,6 +439,31 @@
 		name = "empty scroll"
 		icon_state = "blankscroll"
 
+/obj/item/book/granter/martial/pacifist
+	martial = /datum/martial_art/pacifist
+	name = "strange scroll"
+	martialname = "paci-fist"
+	desc = "A strange scroll filled with martial lessons. It seems to be drawings of some sort of pacifist martial art."
+	greet = "<span class='sciradio'>You have learned the ancient martial art of the Paci-Fist! Your non-harmful abilities has become much more effective, and you are now able to block most attacks \
+	directed toward you. You are also able to nerve pinch an opponent to sleep. You can learn more about your newfound art by using the Recall Teachings verb in the Sleeping Carp tab.</span>"
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "scroll2"
+	remarks = list("I must prove myself worthy to the masters of the paci-fist...", "Stance means everything...", "Focus... And you'll be able to incapacitate any foe in seconds...", "I must avoid conflict for maximum efficiency...", "I don't think this would combine with other martial arts...", "Grab them first and pacify them...", "I must prove myself worthy of this power...")
+
+/obj/item/book/granter/martial/pacifist/already_known(mob/user)
+	if(HAS_TRAIT(user, TRAIT_PACIFISM))
+		return FALSE
+	else
+		to_chat(user, "<span class='warning'>You try to read the scroll but only a real pacifist would probably understand it.</span>")
+		return TRUE
```

Comment:
Doesn't it do same thing as `can_use()`?

Reply:
Practically yes, but you really want me to call on /datum/martial_art/pacifist/can_use() from here?

Reply:
The name suggests this checks if you already know the martial arts, this should be checking if trait has this art's trait.

Reply:
Yeah sorry I forgot to add a . = ..()
But some other martial arts use this proc in much the same manner, since it's just a generic precheck from parent datum really